frontend/src/components/PageHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import { FunctionComponent, ReactNode } from 'react'
+import { Link } from 'react-router-dom'
+import DistributeAidLogo from './DistributeAidLogo'
+
+interface Props {
+  title?: ReactNode
+}
+
+/**
+ * A full-width element with branding and important links.
+ */
+const PageHeader: FunctionComponent<Props> = ({ title }) => {
+  return (
+    <header className="py-2" style={{ background: '#112C71' }}>
```

Comment:
suggestion (non-blocking): extract the color value to a common constants file?

Reply:
Yeah I went back and forth on that, since we don't haven't fleshed out the color palette. A variable is better though 👍